### DISCUSSION TITLE
Command class changes

### DISCUSSION THREAD

**Discussion by fabnavigator** [zwave-js/node-red-contrib-zwave-js#251] [Q&A] [answered]
My garage door tilt sensor ```https://devices.zwave-js.io/?jumpTo=0x014a:0x0004:0x0003:0.0``` event has switched from a command class of Binary Sensor to Notification in the beta release. Needless to say this breaks my code. I scanned the release notes and don't recall seeing this for my device. Did I miss it?

**Comment by marcus-j-davies** [comment, marked as answer]
The NR module does not get involved with what is sent - anything sent is direct from the driver.
unless @AlCalzone  has a quick answer, you need to log this please on the Drivers repo
(and please always supply logs before we can answer anything)

Although check your settings, as clearly the device has settings for this.

![image](https://user-images.githubusercontent.com/55892693/184552530-3532f82c-289d-40f1-8415-55f6e9ae35a2.png)

I'm not going to be around for a couple of weeks.

**Reply by AlCalzone** [reply]
The driver simply forwards/translates what is sent from the device. Binary Sensor reports are not turned into notifactions in any way.

**Reply by fabnavigator** [reply]
@marcus-j-davies Thank you. That was the issue.
